ark/utils/load_utils.py

```diff
@@ -35,9 +35,15 @@ def load_imgs_from_mibitiff(data_dir, mibitiff_files=None, channels=None, delimi
             xarray with shape [fovs, x_dim, y_dim, channels]
     """
 
+    if not os.path.isdir(data_dir):
```

Comment:
We should use validate_path here instead, since it produces a more verbose error message.